com.is_slick.rmac.rmac.metainfo.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="console-application">
+  <id>com.is_slick.rmac.rmac</id>
+  
+  <name>rmac</name>
+  <summary>Assembler for 680x0 processors (and others found in Atari machines)</summary>
+  
+  <metadata_license>MIT</metadata_license>
+  <project_license>MIT</project_license>
```

Comment:
Where is the source code hosted at?

Reply:
When I build the release packages I test tem against a regression suite which is stored privately as it contains copyrighted code. Otherwise I cannot guarantee that the binaries created (by third parties) are correct.

Reply:
>  as it contains copyrighted code

This is also copyrighted code. I guess you mean something else.

Reply:
Yes, I meant about the test suite. Simply put, some people have entrusted me with source code that is to be used internally for regression testing but are not fine with it released publically.

Reply:
> Where is the source code hosted at?

Didn't get an answer to this?

Reply:
Sorry for late response, source is located at http://shamusworld.gotdns.org/cgi-bin/gitweb.cgi?p=rmac;a=tree;h=234f42879ceda29cd6a37a50ff287730f4ac632e;hb=HEAD with backup at http://tiddly.mooo.com:5000/rmac/rmac

Reply:
You should add that repo as vcs-browser in metainfo https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines/#url